bin/build_robots.js

```diff
@@ -0,0 +1,28 @@
+const fs = require('fs');
+const env = process.env.NODE_ENV;
+const directory = './public';
+const filename = 'robots.txt';
+
+const PROD_CONTENT = `User-agent: *
+Allow: /
+Disallow: /search
+Disallow: /alliancemine
+Disallow: /jbrowse
+Sitemap: https://www.alliancegenome.org/sitemap.xml
+Sitemap: https://www.alliancegenome.org/api/sitemap.xml`;
+
+const OTHER_CONTENT = `# https://www.robotstxt.org/robotstxt.html
+User-agent: *
+Disallow:
```

Comment:
I think this should be 
```
User-agent: *
Disallow: /
```
I know thats not what is on stage but reading this site: https://searchfacts.com/robots-txt-allow-disallow-all/

Reply:
Ah, sure. I can do that.